include/configuration_server.hrl

```diff
@@ -0,0 +1,18 @@
+%%%-------------------------------------------------------------------
+%%% @doc
+%%% This header file defines a few macros, that TEP needs to find and
+%%% access a configuration file.
+%%% @end
+%%%-------------------------------------------------------------------
+%% @doc This macro defines the accepted file names for the configuration files.
```

Comment:
```suggestion

%% @doc This macro defines the accepted file names for the configuration files.
```
for readability

Reply:
This will be automatically removed by the formatter `rebar3 fmt`.

Reply:
I see. In that case I guess we'll leave it for now and try to grow accustomed, since it's so opinionated. I could see this becoming annoying however...